lighthouse-core/test/report/report-generator-test.js

```diff
@@ -87,6 +87,8 @@ describe('Report', () => {
     assert.ok(html.includes('data-action="save-json" '), 'has save json button');
     assert.ok(html.includes('class="report__icon copy"'), 'has copy button');
     assert.ok(html.includes('class="report__icon print"'), 'has print button');
+    assert.ok(html.includes(
```

Comment:
added to cover https://github.com/GoogleChrome/lighthouse/pull/1790#issuecomment-282894087 in the future.